src/analysis/retail/shaman/restoration/modules/Abilities.ts

```diff
@@ -28,7 +28,10 @@ class Abilities extends CoreAbilities {
         spell: TALENTS.RIPTIDE_TALENT.id,
         category: SPELL_CATEGORY.ROTATIONAL,
         enabled: combatant.hasTalent(TALENTS.RIPTIDE_TALENT),
-        charges: combatant.hasTalent(TALENTS.ECHO_OF_THE_ELEMENTS_TALENT) ? 2 : 1,
+        charges:
+          1 +
+          (combatant.hasTalent(TALENTS.ECHO_OF_THE_ELEMENTS_TALENT) ? 1 : 0) +
+          (combatant.hasTalent(TALENTS.ELEMENTAL_REVERB_TALENT) ? 1 : 0),
```

Comment:
this can be simplified to:
```suggestion
        charges:
          1 + combatant.getTalentRank(TALENTS.ECHO_OF_THE_ELEMENTS_TALENT) +
          combatant.getTalentRank(TALENTS.ELEMENTAL_REVERB_TALENT),
```